files/similar/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "similar",
+  "repo": "InputSimilar/similar",
+  "files": {
+    "include": ["similar.eva.js"]
```

Comment:
I am saying that the bot uses this directory as source https://github.com/InputSimilar/similar/tree/1.1.0
And there is no `similar.eva.js` in there. So the bot will fail